eslint.config.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * @fileoverview ESLint configuration file
+ * @author Nicholas C. Zakas
+ */
+
+//-----------------------------------------------------------------------------
+// Imports
+//-----------------------------------------------------------------------------
+
+import eslintConfigESLint from "eslint-config-eslint";
+import n from "eslint-plugin-n";
+//-----------------------------------------------------------------------------
+// Config
+//-----------------------------------------------------------------------------
+
+export default [
+
+    {
+        ignores: [
+            "tests/fixtures",
+            "coverage",
+            "docs",
+            "jsdoc",
+            "dist",
+        ]
+    },
+
+    ...eslintConfigESLint,
```

Comment:
We should also enable [Formatting config](https://github.com/eslint/eslint/tree/main/packages/eslint-config-eslint#formatting-config), for now.